packages/model-viewer/src/three-components/gltf-instance/correlated-scene-graph.ts

```diff
@@ -60,47 +60,62 @@ export class CorrelatedSceneGraph {
       CorrelatedSceneGraph {
     const gltf = threeGLTF.parser.json as GLTF;
 
-    const {associations} = threeGLTF.parser;
+    const associations =
+        threeGLTF.parser.associations as Map<ThreeSceneObject, GLTFReference>;
     const gltfElementMap: GLTFElementToThreeObjectMap = new Map();
 
     const defaultMaterial = {name: 'Default'} as Material;
-    const defaultReference = {type: 'materials', index: -1} as GLTFReference;
-
-    // NOTE: IE11 does not have Map iterator methods
-    associations.forEach((gltfElementReference, threeObject) => {
-      // Note: GLTFLoader creates a "default" material that has no corresponding
-      // glTF element in the case that no materials are specified in the source
-      // glTF. In this case we append a default material to allow this to be
-      // operated upon.
-      if (gltfElementReference == null) {
+    const defaultReference = {type: 'materials', index: -1};
+
+    for (const threeMaterial of associations.keys()) {
+      // Note: GLTFLoader creates a "default" material that has no
+      // corresponding glTF element in the case that no materials are
+      // specified in the source glTF. In this case we append a default
+      // material to allow this to be operated upon.
```

Comment:
Do we have a test for loading a glTF with no  material? Might be good to check that this gets created properly.

Reply:
correlated-scene-graph does cover the default material, but I will add an e2e test covering a primitives configuration of the default material.